spark/src/main/scala/io/delta/sharing/spark/RemoteDeltaLog.scala

```diff
@@ -171,7 +184,8 @@ private[sharing] object RemoteDeltaLog {
           java.lang.Boolean.valueOf(sslTrustAll),
           java.lang.Boolean.valueOf(forStreaming))
         .asInstanceOf[DeltaSharingClient]
-    new RemoteDeltaLog(deltaSharingTable, new Path(path), client)
+
+    new RemoteDeltaLog(deltaSharingTable, new Path(path + getFormattedTimestampWithUUID), client)
```

Comment:
Do we need a unit test?

Reply:
let me see if I can add one. unit test should be just for the suffix, the functionality of e2e query is covered by existing tests.

Reply:
added